src/main/java/org/apache/ibatis/builder/MapperBuilderAssistant.java

```diff
@@ -337,7 +337,7 @@ private ParameterMap getStatementParameterMap(
         try {
           resultMaps.add(configuration.getResultMap(resultMapName.trim()));
         } catch (IllegalArgumentException e) {
-          throw new IncompleteElementException("Could not find result map " + resultMapName, e);
+          throw new IncompleteElementException("Could not find result map " + resultMapName + " [statementId]" + statementId, e);
```

Comment:
Please change the error message a little bit.

```suggestion
          throw new IncompleteElementException("Could not find result map '" + resultMapName + "' referenced from '" + statementId + "'", e);
```